{
  "uid" : "ffe790a2087c61a1",
  "name" : "reverts when trying to transferFrom",
  "fullName" : "Contract: ERC721Pausable when token is paused.reverts when trying to transferFrom",
  "historyId" : "f7988b9086dea19e74d2f8b1239f2b1b",
  "time" : {
    "start" : 1635555013426,
    "stop" : 1635555014397,
    "duration" : 971
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7aa90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7aa90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC721/extensions/ERC721Pausable.test.js:15:46)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ERC721Pausable when token is paused"
  }, {
    "name" : "testClass",
    "value" : "Contract: ERC721Pausable when token is paused"
  }, {
    "name" : "testMethod",
    "value" : "reverts when trying to transferFrom"
  }, {
    "name" : "package",
    "value" : "Contract: ERC721Pausable when token is paused"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "e8620166440a2229",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a043eca0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554949660,
        "stop" : 1635554950662,
        "duration" : 1002
      }
    }, {
      "uid" : "e7b77bbcc1b3876a",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07f5820>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554925223,
        "stop" : 1635554925814,
        "duration" : 591
      }
    }, {
      "uid" : "1d65acb0af833410",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a05107c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554914107,
        "stop" : 1635554914699,
        "duration" : 592
      }
    }, {
      "uid" : "4d29a8e1551a9321",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067eb80>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554909438,
        "stop" : 1635554910133,
        "duration" : 695
      }
    }, {
      "uid" : "ae142b5b45f6ddb7",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07c3d60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554877337,
        "stop" : 1635554878339,
        "duration" : 1002
      }
    }, {
      "uid" : "873fc157cedfeb47",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0428220>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554845351,
        "stop" : 1635554845943,
        "duration" : 592
      }
    }, {
      "uid" : "555ae9f41a3e7cf5",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067ca30>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554832693,
        "stop" : 1635554833284,
        "duration" : 591
      }
    }, {
      "uid" : "a5ec59a12e32d14e",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0673fd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554829559,
        "stop" : 1635554830151,
        "duration" : 592
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "1b55da495e2387f6",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/1b55da495e2387f6",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551256189,
          "stop" : 1635551256249,
          "duration" : 60
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "ffe790a2087c61a1.json",
  "parameterValues" : [ ]
}